src/ui/CategoryFacet/CategoryFacet.ts

```diff
@@ -529,6 +554,11 @@ export class CategoryFacet extends Component implements IAutoLayoutAdjustableIns
     }
   }
 
+  public triggerNewQuery(beforeExecuteQuery?: () => void) {
```

Comment:
🤔 It seems like it's a weird way to make it so you can decide to "log a analytics event, or supress the warnings that comes with it otherwise".

Is there a real reason why we would want that ? 
Is that some kind of artifact from the past version which we don't need anymore ?

Reply:
I don't think so.. it's still active in the QueryController.  I remember seeing a warning that's why it's there for the DynamicFacet as well.
> If the analytics component is enabled in the interface, it will look for any query executed by the query controller for which no analytics event was associated.<br/>
> By setting this property to true, this will cancel this check when the query is performed